File: week-05/day-05/src/NthElement/NthElement.cs
using System;

namespace FulvipesFoundationExamRetake
{
    public class NthElement
    {
        public static void Main(string[] args)
        {
            int[] numbers = new int[] {2, 4, 10, 34, 3, 16, 3, 21};
            int n = 2;

            double result = GetAverageForNthElements(numbers, n);
            Console.WriteLine("The average for every {0} element is {1}", n, result);
        }

    }
}

File: week-05/day-05/src/NthElement/NthElement.java
package NthElement;

public class NthElement {

  public static double nthElementAvg(int[] numbers, int n) {
    double counter = 0;
    double sum = 0;
    int position = n;
    if (numbers.length >= n) {
      for (int i = 0; i < numbers.length; i++) {
        if (i == position - 1) {
          counter++;
          sum += numbers[i];
          position += n;
        }
      }
      return sum / counter;
    } else {
      return 0;
    }
  }
}
